.gitignore

```diff
@@ -60,3 +60,6 @@ target/
 
 #Ipython Notebook
 .ipynb_checkpoints
+
+#PyPI Configuration
+.pypirc
```

Comment:
This file should be in your home directory, does it read from the local dir also?